.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@korsvanloon What we discussed last week about removing this, should we go through with it in our open source packages as well? It feels like the best place to start enforcing more of the standard rules.